Software/Python/gopigo.py

```diff
@@ -43,6 +43,24 @@
 import struct
 import subprocess
 
+# Test GoPiGo vs GoPiGo3
+# If
+try:
+    import gopigo3
+    try:
```

Comment:
So let's get it straight:
* If the `gopigo3` isn't installed, then a message is printed out and that's fine.
* If the `gopigo3` is installed and the user is importing this `gopigo` module, he should at least get a message warning him that he may not be running the right module.
* If the `gopigo3` is installed and instantiating the `GoPiGo3` object fails, then the GoPiGo3 robot isn't available, which is great and so no message should be displayed.

I think these 3 bullet points describe the current situation as it is. Hope I haven't missed anything.

Reply:
Same three points, I think.
1. The gopigo3 library isn't installed and we trust to user to know what they're doing (they're on Raspbian) -> no message
2. The gopigo3 library is installed and the GoPiGo3 can be instantiated => issue a stern message, including a reference to  using `import gopigo3` and a link to the online documentation
3. The gopigo3 library is installed and the GoPiGo3 cannot be instantiated => all's fine, no messages